src/plugins/GUITestBase/src/tests/regression_scenarios/GTTestsRegressionScenarios_6001_7000.cpp

```diff
@@ -6605,6 +6607,52 @@ GUI_TEST_CLASS_DEFINITION(test_6959) {
     CHECK_SET_ERR(nameList3[5] == "Conocephalus_discolor", "The 6 sequence is incorrect");
 }
 
+GUI_TEST_CLASS_DEFINITION(test_6960) {
+    class ProjectPathValidationScenario : public CustomScenario {
+    public:
+        void run(HI::GUITestOpStatus &os) override {
+            GTGlobals::sleep();
```

Comment:
Please never use sleep() in the test unless this is a part of the logic of the test: "like wait for 30 seconds for a long-run task and cancel it"